src/Raven.Server/Integrations/PostgreSQL/RqlQuery.cs

```diff
@@ -17,21 +17,32 @@
 using Raven.Server.Utils;
 using Sparrow.Json;
 using Sparrow.Json.Parsing;
+using Sparrow.Logging;
 
 namespace Raven.Server.Integrations.PostgreSQL
 {
     public class RqlQuery : PgQuery
     {
         protected readonly DocumentDatabase DocumentDatabase;
-        private readonly QueryOperationContext _queryOperationContext;
+        private QueryOperationContext _queryOperationContext;
         private List<Document> _result;
         private readonly int? _limit;
+        private bool _queryWasRun;
+        private static readonly Logger Logger = LoggingSource.Instance.GetLogger<PgSession>("Postgres RqlQuery");
 
+        ~RqlQuery()
+        {
+            Logger.Operations($"Query '{this.QueryString}' wasn't disposed properly.\n" +
```

Comment:
Please add `if (Logger.IsOperationsEnabled)`

Reply:
Instead of `\n` it's better to use `Environment.NewLine` so it will be formatted correctly on Windows too